Azure-Samples/chat-with-your-data-solution-accelerator
# Move python tests to root

## Purpose
Pull the Unit Tests into a folder from the root so that we don't deploy them

Fixes #200 

Most tests are disabled as they require Azure Services. We can enable them once we have CD. This is an interim PR so that we can ask contributors to add tests and know that they will be run.

## Does this introduce a breaking change?
```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?
```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[x] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
Unit Tests should run from a PR with this change 🤞

![image](https://github.com/Azure-Samples/chat-with-your-data-solution-accelerator/assets/6254153/7e4345bb-a2fe-4255-82e3-59df259d93e6)
